docs/config/index.md

```diff
@@ -382,6 +382,18 @@ To get TypeScript working with the global APIs, add `vitest/globals` to the `typ
 }
 ```
 
+Sometimes the configuration mentioned above might not work because your workspace cannot find the types for `vitest/globals`. If that's the case, you can add `typeRoots` to your `tsconfig.json` file. Afterward, try restarting your TypeScript server. For example in VSCode, press `F1` and select `TypeScript: Restart TS server`
```

Comment:
I'm not sure if it's Vitest's documentation's place to instruct restarting IDEs Typescript servers. It's IDE specific.

Reply:
You are true @AriPerkkio , I think VSCode having a problem to recognize vitest/globals, adding that steps make it works but after for some times I can use vitest/globals without typeRoots and restarting VSCode TS server.

Reply:
Any suggestions regarding that?

Reply:
Is it really Vitest specific? Or is VSCode just not automatically restarting the Typescript server after `tsconfig.json` was modified?